datadog_checks_dev/tests/conftest.py

```diff
@@ -27,3 +29,6 @@ def mock_e2e_metadata():
 @pytest.fixture(scope='session')
 def dd_environment(mock_e2e_config, mock_e2e_metadata):
     yield mock_e2e_config, mock_e2e_metadata
+
+
+not_appveyor = pytest.mark.skipif(running_on_appveyor(), reason="Test can't be run on Appveyor")
```

Comment:
Can we move this to `tests/utils.py`? I've seen issues with importing from conftest directly.